src/bos/server/controllers/v2/sessiontemplates.py

```diff
@@ -91,6 +92,11 @@ def _validate_sanitize_session_template(session_template_id, template_data):
             raise ParsingException(f"boot_sets key ({bs_name}) does not match 'name' "
                                    f"field of corresponding boot set ({bs["name"]})")
 
+        # Also, validate that each boot set has at least one of the BOOT_SET_NODE_FIELDS
+        if any(field_name in bs for field_name in BOOT_SET_NODE_FIELDS):
```

Comment:
I feel like this construct is slightly dangerous to future coders.

if <good things happens>:
  goto top of loop
SCREAM, "Badness ensued!!! Eject! Eject!"

If I miss that goto (i.e. continue), I might put stuff after it.

Can you do
if not any(...):
    raise ParsingException(f"Boot set {bs_name} has none of the following fields: {BOOT_SET_NODE_FIELDS}")
instead?

Reply:
Sure thing. I did it this way because of a recent guide I had read which suggested that "if not" is to be avoided in general, but I don't feel strongly about it.